plugins/modules/k8s.py

```diff
@@ -472,6 +480,7 @@ def argspec():
         type="dict", default=None, options=server_apply_spec()
     )
     argument_spec["delete_all"] = dict(type="bool", default=False, aliases=["all"])
+    argument_spec["hidden_fields"] = dict(type="list", elements="str")
```

Comment:
```suggestion
    argument_spec["hidden_fields"] = dict(type="list", elements="str", default=[])
```

Reply:
I don't think this is necessary - it handles the implicit `default=None` just fine